exchanges/binanceus/binanceus_test.go

```diff
@@ -0,0 +1,1929 @@
+	if !errors.Is(er, errIncompleteArguments) {
+		t.Errorf("Binanceus GetOCOOrder() error  expecting %v, but found %v", errIncompleteArguments, er)
+	}
+	_, er = bi.GetOCOOrder(context.Background(), &GetOCOOrderRequestParams{
+		OrderListID: "123445",
+	})
+	if er != nil && !strings.Contains(er.Error(), "Order list does not exist.") {
+		t.Error("Binanceus GetOCOOrder() error", er)
+	}
+}
+
+func TestGetAllOCOOrder(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	_, er := bi.GetAllOCOOrder(context.Background(), &OCOOrdersRequestParams{})
+	if er != nil {
+		t.Error("Binanceus GetAllOCOOrder() error", er)
+	}
+}
+
+func TestGetOpenOCOOrders(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	_, er := bi.GetOpenOCOOrders(context.Background(), 0)
+	if er != nil {
+		t.Error("Binanceus GetOpenOCOOrders() error", er)
+	}
+}
+
+func TestCancelOCOOrder(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() || !canManipulateRealOrders {
+		t.SkipNow()
+	}
+	_, er := bi.CancelOCOOrder(context.Background(), &OCOOrdersDeleteRequestParams{})
+	if !errors.Is(er, errIncompleteArguments) {
+		t.Errorf("Binanceus CancelOCOOrder() error expected %v, but found %v", errIncompleteArguments, er)
+	}
+}
+
+// OTC end Points test code.
+func TestGetSupportedCoinPairs(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	_, er := bi.GetSupportedCoinPairs(context.Background(), currency.Pair{Base: currency.BTC, Quote: currency.USDT})
+	if er != nil {
+		t.Error("Binanceus GetSupportedCoinPairs() error", er)
+	}
+}
+
+func TestRequestForQuote(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.SkipNow()
+	}
+	_, er := bi.RequestForQuote(context.Background(), &RequestQuoteParams{ToCoin: "BTC", RequestCoin: "USDT", RequestAmount: 1})
+	if er != nil && !errors.Is(er, errMissingFromCoinName) {
+		t.Errorf("Binanceus RequestForQuote() expecting %v, but found %v", errMissingFromCoinName, er)
+	}
+	_, er = bi.RequestForQuote(context.Background(), &RequestQuoteParams{FromCoin: "ETH", RequestCoin: "USDT", RequestAmount: 1})
+	if er != nil && !errors.Is(er, errMissingToCoinName) {
+		t.Errorf("Binanceus RequestForQuote() expecting %v, but found %v", errMissingToCoinName, er)
+	}
+	_, er = bi.RequestForQuote(context.Background(), &RequestQuoteParams{FromCoin: "ETH", ToCoin: "BTC", RequestCoin: "USDT"})
+	if er != nil && !errors.Is(er, errMissingRequestAmount) {
+		t.Errorf("Binanceus RequestForQuote() expecting %v, but found %v", errMissingRequestAmount, er)
+	}
+	_, er = bi.RequestForQuote(context.Background(), &RequestQuoteParams{FromCoin: "ETH", ToCoin: "BTC", RequestAmount: 1})
+	if er != nil && !errors.Is(er, errMissingRequestCoin) {
+		t.Errorf("Binanceus RequestForQuote() expecting %v, but found %v", errMissingRequestCoin, er)
+	}
+	_, er = bi.RequestForQuote(context.Background(), &RequestQuoteParams{FromCoin: "BTC", ToCoin: "USDT", RequestCoin: "BTC", RequestAmount: 1})
```

Comment:
```json
Binanceus raw response: {"symbol":"BTCUSDT","ratio":"22129.5","inverseRatio":"0.00004519","validTimestamp":1658714797098,"toAmount":"22129.5","fromAmount":"1"}
```

For some reason quoteID is not coming back, we might be able to remove the field from Quote type?